Sources/AudioKit/Operations/AKOperation.swift

```diff
@@ -103,7 +103,7 @@ open class AKOperation: AKComputedParameter {
     public static var rightInput = AKOperation("(15 p) ")
 
     /// Dummy trigger
-    public static var trigger = AKOperation("(0 p) ")
+    public static var trigger = AKOperation("(14 p) ")
```

Comment:
I have no record of any playground or project actually using trigger. From the code it looks like all of the parameters could be triggers. But only on generators. Its kind of a mess so what I'm planning to do is make examples and tests to solidify what are parameters and what are triggers. Perhaps there can be 7 of each instead of 14 multipurpose ones.